content/en/ensign/index.md

```diff
@@ -1,20 +1,38 @@
 ---
 date: 2022-12-15T10:09:42Z
-title: "Ensign for Data Teams"
-headertext: "Data Science 2.0"
-subheadertext: "Streaming Predictions for Realtime Models"
+title: "Ensign Use Cases"
+headertext: "Ensign Use Cases"
 noicon: true
 type: "ensign"
 switch:
     link: /ensign-enterprise/
     text: "Learn about Ensign for Enterprise"
 ---
 
-## Think Outside the Batch
+## Use Case 1: Turn Your Model into an API
+Data science teams are realizing that they must be in charge of deploying and managing their models. Sound controversial? Check out [this article](https://venturebeat.com/ai/why-do-87-of-data-science-projects-never-make-it-into-production/) about why 87% of data science projects never make it to production.
```

Comment:
None of the links will open in a new tab. I wonder if we should create a story to use a shortcode for links in `.md` files so that they open in new tabs. Hugo may have built-in one for this.